### PR TITLE
Create Play Styles Stubs

### PR DESCRIPTION
**Pull request by paper-toaster**
Resolves part of #3149; part of overall organization effort #3143. 

Creates stubs for the following topics: 
- Single-tapping
- Alternating
- Double Tapping
- Hybrid
- Tapx 
- Tapping style

### CODE REVIEW

File: wiki/Play_Styles/Alternating/en.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+tags:
+  - keyboard
+  - tap
+  - alternate
+  - alt
+  - play style
+---
+
+<!-- This article is a stub -->
```
Comment (paper-toaster) [inline wiki/Play_Styles/Alternating/en.md:10]:
```suggestion
---
stub: true
tags:
  - keyboard
  - tap
  - alternate
  - alt
  - play style
---
```

File: wiki/Play_Styles/Double_Tapping/en.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+tags:
+  - keyboard
+  - tap
+  - hybrid
+  - play style
+---
+
+<!-- This article is a stub -->
```
Comment (paper-toaster) [inline wiki/Play_Styles/Double_Tapping/en.md:9]:
```suggestion
---
stub: true
tags:
  - keyboard
  - tap
  - hybrid
  - play style
---
```

File: wiki/Play_Styles/Hybrid/en.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+tags:
+  - keyboard
+  - tap
+  - hybrid
+  - play style
+---
+
+<!-- This article is a stub -->
```
Comment (paper-toaster) [inline wiki/Play_Styles/Hybrid/en.md:9]:
```suggestion
---
stub: true
tags:
  - keyboard
  - tap
  - hybrid
  - play style
---
```

File: wiki/Play_Styles/Singletapping/en.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+tags:
+  - keyboard
+  - tap
+  - single tap
+  - play style
+---
+
+<!-- This article is a stub -->
```
Comment (paper-toaster) [inline wiki/Play_Styles/Singletapping/en.md:9]:
```suggestion
---
stub: true
tags:
  - keyboard
  - tap
  - single tap
  - play style
---
```

File: wiki/Play_Styles/Tapx/en.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+---
+tags:
+  - tablet tap
+  - tap
+  - tablet
+  - play style
+---
+
+<!-- This article is a stub -->
```
Comment (paper-toaster) [inline wiki/Play_Styles/Tapx/en.md:9]:
```suggestion
---
stub: true
tags:
  - tablet tap
  - tap
  - tablet
  - play style
---
```

File: wiki/Play_Styles/Alternating/en.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  - play style
+---
+
+# Alternating
+
+Alternating is a tapping style for [osu!standard](/wiki/Game_Modes/osu!) in which players click circles, slide sliders, and spin spinners by alternating between two set keys on their keyboard (`Z` and `X` by default) every other [hit object](/wiki/Hit_object) or so. Hence the name. This style is most common amongst more seasoned players and is one of the hardest to adapt to.
+
+## Usage
+
+Alternating has a wide variety of uses and it's forms can vary from player to player. For example, many players who alternate typically switch between `Z` and `X` on every other note (or hit object), while others switch keys after every two to three hit objects. It should be noted that some players will alternate most of the time, but sometimes hit the same key twice if a hit object catches them off-guard. This is all simply down to personal preference.
```
Comment (MegaApplePi) [inline wiki/Play_Styles/Alternating/en.md:17]:
```suggestion
Alternating has a wide variety of uses and its forms can vary from player to player. For example, many players who alternate typically switch between `Z` and `X` on every other note (or hit object), while others switch keys after every two to three hit objects. It should be noted that some players will alternate most of the time, but sometimes hit the same key twice if a hit object catches them off-guard. This is all simply down to personal preference.
```
I'm not exactly sure, is this right?

**Pull request merged**